### PR TITLE
task was done

### PR DESCRIPTION
**Pull request by vanyachyzh**
- [DEMO LINK](https://vanyachyzh.github.io/js_growth_table_DOM/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,76 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table').tBodies[0];
+const addRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const addColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const minLimit = 2;
+const maxLimit = 10;
```
Comment (VitaliyBondarenko1982) [inline src/scripts/main.js:9]: constants should be uppercase

Comment (VitaliyBondarenko1982) [reply src/scripts/main.js:9]: Please check all places

**Review by VitaliyBondarenko1982 — APPROVED** [review]
Nice job.
Just pay attention on naming

File: src/scripts/main.js (hunk 1 above)
Comment (vanyachyzh) [reply src/scripts/main.js]: Sorry I didn't have to request again

**Pull request closed without merging**